runner/fake/fake.go

```diff
@@ -11,12 +11,23 @@ func NewRunner() runner.Runner {
 type fakeRunner struct {
```

Comment:
Huh, maybe we don't need fake runner at all now that we have the simple runner and the sim execer?